### ISSUE TITLE
Chat confirmation buttons should be a toolbar for accessibility

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#275893]
Ref https://github.com/microsoft/vscode/issues/275233#event-20710726573
